examples/multifit_nlinear.rs

```diff
@@ -0,0 +1,88 @@
+extern crate rgsl;
+
+use rgsl::gsl_multifit_nlinear_basic;
+use rgsl::gsl_multifit_nlinear_basic_df;
+
+use rgsl::types::rng::Rng;
+use rgsl::types::rng::RngType;
+
+
+#[no_mangle]
```

Comment:
There is no need or `no_mangle` in here.

Reply:
This has been removed.